[2025-03-17T17:02:45.426+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-17T17:02:45.436+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.first_task scheduled__2011-04-03T00:00:00+00:00 [queued]>
[2025-03-17T17:02:45.442+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.first_task scheduled__2011-04-03T00:00:00+00:00 [queued]>
[2025-03-17T17:02:45.442+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-17T17:02:45.450+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2011-04-03 00:00:00+00:00
[2025-03-17T17:02:45.455+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=474) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-17T17:02:45.456+0000] {standard_task_runner.py:72} INFO - Started process 483 to run task
[2025-03-17T17:02:45.456+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'first_task', 'scheduled__2011-04-03T00:00:00+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/fisrt_dag_1.py', '--cfg-path', '/tmp/tmp9zjyv94o']
[2025-03-17T17:02:45.457+0000] {standard_task_runner.py:105} INFO - Job 51: Subtask first_task
[2025-03-17T17:02:45.482+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.first_task scheduled__2011-04-03T00:00:00+00:00 [running]> on host 244d68fe4748
[2025-03-17T17:02:45.523+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2011-04-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2011-04-03T00:00:00+00:00'
[2025-03-17T17:02:45.523+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-17T17:02:45.532+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-17T17:02:45.532+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world']
[2025-03-17T17:02:45.538+0000] {subprocess.py:99} INFO - Output:
[2025-03-17T17:02:45.538+0000] {subprocess.py:106} INFO - hello world
[2025-03-17T17:02:45.539+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-17T17:02:45.554+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-17T17:02:45.555+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=first_task, run_id=scheduled__2011-04-03T00:00:00+00:00, execution_date=20110403T000000, start_date=20250317T170245, end_date=20250317T170245
[2025-03-17T17:02:45.597+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-17T17:02:45.606+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-17T17:02:45.607+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
